### DISCUSSION TITLE
Custom scrollbar with QTable in virtual-scroll mode ?

### DISCUSSION THREAD

**Discussion by squat80** [quasarframework/quasar#11542] [General - Components / Directives / etc] [answered]
Hi

i'm using quasar 2.3.1 and have been playing with qtable and virtual-scroll

is there any way to have "beautiful" scrollbar like i would with q-scroll-area in the qtable virtualscroll ?
![image](https://user-images.githubusercontent.com/1357779/144471853-19ed2dba-69ed-4b65-83eb-7f2233176204.png)

right now i seem to only be able to use the browser native scrollbars...
![image](https://user-images.githubusercontent.com/1357779/144471683-037c2913-6bd8-4a8d-89d0-8a47cf199132.png)

i searched the online document as well as discussions and issues but i did not find any clues (sorry if i missed them)

thanks for your advices
David

**Comment by MilosPaunovic** [comment, marked as answer]
Take a look at this; it might solve your problems:
[https://github.com/quasarframework/quasar/discussions/10087](https://github.com/quasarframework/quasar/discussions/10087)
